app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -279,23 +269,11 @@ class EventDetailsFragment : Fragment() {
     private fun loadEvent(event: Event) {
         val startsAt = EventUtils.getEventDateTime(event.startsAt, event.timezone)
         val endsAt = EventUtils.getEventDateTime(event.endsAt, event.timezone)
-
-        rootView.eventName.text = event.name
+        binding.setVariable(BR.event, event)
```

Comment:
binding.setEvent

Reply:
@iamareebjamal updated with the required changes should i continue this PR now for recyclerview and other fragments